### PR TITLE
See an individual reminder

### PR DESCRIPTION
**Pull request by hmorri32**
dynamic path to reminders/reminder/id
active class styling
defined routes
tests pass

## Purpose

dynamic routing to individual reminder

## Approach

used link-to in order to route to individual reminder

### Learning

consulted the ember docs on active classnames as well as dynamic, nested routing. 

### Open Questions and Pre-Merge TODOs

- [ ] Cool!

### Test coverage 

acceptance test for URL change as well as rendering title of reminder

### Follow-up tasks

- [Issue #4 (Example)](https://github.com/flexyford/pull-request/issues)

### Screenshots
Screenshot

### CODE REVIEW

**Comment by hmorri32** [comment]
@Tman22 @martensonbj

**Comment by Tman22** [comment]
Hey I appreciate you doing the recordit!

File: app/routes/reminders/reminder.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import Ember from 'ember';
+
+export default Ember.Route.extend({
+  model(params) {
+    return this.store.findRecord('reminder', params.id)
```
Comment (Tman22) [inline app/routes/reminders/reminder.js]: Be consistent on semi-colons.

File: tests/unit/routes/reminders/reminder-test.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import { moduleFor, test } from 'ember-qunit';
+
+moduleFor('route:reminders/reminder', 'Unit | Route | reminders/reminder', {
+  // Specify the other units that are required for this test.
+  // needs: ['controller:foo']
+});
```
Comment (Tman22) [inline tests/unit/routes/reminders/reminder-test.js]: Delete this

File: app/routes/reminders/reminder.js (hunk 1 above)
Comment (hmorri32) [reply app/routes/reminders/reminder.js]: ya... @jonwille

Comment (ActionJonny) [reply app/routes/reminders/reminder.js]: I will try and get better

File: tests/unit/routes/reminders/reminder-test.js (hunk 2 above)
Comment (martensonbj) [reply tests/unit/routes/reminders/reminder-test.js]: (He means the whole file)

**Comment by hmorri32** [comment]
@Tman22 @martensonbj Deleted obsolete tests, fixed jons inconsistent semicolon usage

**Comment by martensonbj** [comment]
👍

**Comment by hmorri32** [comment]
close #3

**Pull request merged**